tests/golos.vesting_test_api.hpp

```diff
@@ -84,20 +84,67 @@ struct golos_vesting_api: base_contract_api {
             ("symbol", sym)
         );
     }
-
-    action_result delegate(name from, name to, asset quantity,
-        uint16_t interest_rate = 0
-    ) {
-        return push(N(delegate), from, args()
-            ("from", from)
-            ("to", to)
-            ("quantity", quantity)
-            ("interest_rate", interest_rate)
+    
+    std::vector<variant> get_delegators() {
+        return _tester->get_all_chaindb_rows(_code, _symbol.to_symbol_code().value, N(delegation), false);
+    }
+    
+    int64_t get_delegated(name from, name to) {
+        auto delegators = get_delegators();
+        for (auto& d : delegators) {
+            if (d["delegator"].as<name>() == from && d["delegatee"].as<name>() == to) {
+                return d["quantity"].as<asset>().get_amount();
+            }
+        }
+        return 0;
+    }
+    
+     action_result delegate_unauthorized(name from, name to, asset quantity, bool by_delegatee) {
+        return push(N(delegate), by_delegatee ? from : to, args() 
+            ("from", from) 
+            ("to", to) 
+            ("quantity", quantity) 
+            ("interest_rate", 0) 
         );
+    } 
+
+    action_result delegate(name from, name to, asset quantity, uint16_t interest_rate = 0) {
```

Comment:
please move this to something like `delegate_with_cyber_msig` and use simplified `delegate` with `push_msig()` inside to provide several signatures